tests/test_migrate_coverage.py

```
@@ -0,0 +1,1092 @@
+                name (str): Name of the module or attribute to import; forwarded to the real import.
+                *args: Positional arguments forwarded to the real import.
+                **kwargs: Keyword arguments forwarded to the real import.
+
+            Returns:
+                object: The imported module or attribute as returned by the real import.
+
+            Raises:
+                ImportError: If `name` is equal to "yaml".
+            """
+            if name == "yaml":
+                raise ImportError("Mock import error")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)